src/main/java/org/lobid/resources/Triples2RdfModel.java

```diff
@@ -92,31 +92,30 @@ public void setPropertyIdentifyingTheNodeForInferencing(String property) {
 	public void process(final String str) {
 		if (str.startsWith("<" + PipeEncodeTriples.DUMMY_SUBJECT)) {
 			LOG.warn("Model without subject - skipping. Model size=" + str.length());
-		} else {
-			Model model = ModelFactory.createDefaultModel();
-			try {
-				model.read(new StringReader(str), "test:uri" + count++, serialization);
-				if (inferencing)
-					reasoning(model);
-				getReceiver().process(model);
-			} catch (RiotException rioe) {
-				LOG.warn("Resource with some broken triples:\n" + str);
-				String seb = new BufferedReader(new StringReader(str)).lines()
-						.filter((line) -> validTriple(line, model))
-						.collect(Collectors.joining("\n"));
-				model.read(new StringReader(seb), "test:uri" + count++, serialization);
-				getReceiver().process(model);
-			} catch (Exception e) {
-				LOG.error("Exception in " + str, e);
-			}
-		}
+        }
+        else {
+            Model model = ModelFactory.createDefaultModel();
+            try {
+                String seb = new BufferedReader(new StringReader(str)).lines()
+                    .filter((line) -> validTriple(line, model))
+                    .collect(Collectors.joining("\n"));
+                model.read(new StringReader(seb), "test:uri" + count++, serialization);
+                if (inferencing) {
+                    reasoning(model);
+                }
+                getReceiver().process(model);
+            }
+            catch (Exception e) {
+                LOG.error("Exception in " + str, e);
+            }
+        }
 	}
 
 	private boolean validTriple(String triple, Model model) {
 		try {
 			model.read(new StringReader(triple), "test:uri" + count++, serialization);
 		} catch (Exception e) {
-			LOG.info("... ignore broken triple: " + triple);
+			LOG.info("... ignore broken triple");
```

Comment:
Is this intentional? No details available here?

Reply:
An ErrorHandler changed it's behaviour in newer jena lib: ErrorHandlerFactory writes ERROR messages which hereby will be infoed as to be ignored (tried to configure the ErrorHandler to not be strict but didn't achieve that. Would need some more refactoring. )